CHANGELOG.md

```diff
@@ -29,6 +29,9 @@
 * Resetting Grid columns to their default state (e.g. via the Column Chooser) retains enhancements
   applied from matching Store fields.
 * Desktop `DateInput` now handles out-of-bounds dates without throwing exception during rendering.
+* A grid column header that has a function for headerName argument or element/object for resolved headerName
```

Comment:
too much detail about the technical details of how this workaround happened -- let's just briefly explain what bug fixed